GOVERNANCE.md

```diff
@@ -0,0 +1,74 @@
+# Eclipse iceoryx governance
+
+Eclipse iceoryx is governed democratically by its maintainers under the umbrella of [OpenADx](https://openadx.eclipse.org/).
+This document describes the processes additionally to the [Eclipse process](https://www.eclipse.org/projects/handbook/).
+
+## Maintainers
+
+An up to date list of the maintainers can be found at the [Eclipse project page](https://projects.eclipse.org/projects/technology.iceoryx/who).
+
+## Bi-weekly meetup
+
+Bi-weekly every Thursday for 1 hour. Everyone is welcome to join.
+
+```
+08:00 PST (GMT - 8)
+16:00 GMT
+17:00 CET (GMT + 1)
+21:30 IST (GMT + 5.5)
+23:00 CST (GMT + 7)
+```
+
+### Topics
+
+* Propse features
+  * E.g. present and discuss a design document
+* Discuss pull requests without going into details
+  * E.g. a small demo
+* Steer direction of Eclipse iceoryx
+  * E.g. go over the milestone list and move the issues between the three prio's
+* Raise and answer questions
+* Discuss tooling and infrastructure
+* All others topics related to iceoryx one can think of
+
+> Friendly-reminder:
+>
+> * Discussions about technical issues or in-depth topics should be **avoided**, e.g. "Why is my build broken? I need help"
+> * Please stick to the agenda, stay on the topic and move side topics to other meetings
+
+### Meeting minutes
+
+Meetinges minutes are stored in the [wiki](https://github.com/eclipse/iceoryx/wiki/Developer-meetup).
+An Eclipse maintainer will save the meeting minutes on a new page for every meeting.
+
+If you have points that you want to discuss, please send your points to the [mailing list](mailto:iceoryx-dev@eclipse.org) a couple of days before.
+
+#### Minutes template
+
+```
+Date: 1st January 1900
+Time: 17:00 CET
+
+## Attendees
+
+* Max Mustermann, max.mustermann@mustercompany.com
+* John Doe, john.doe@doecompany.com
```

Comment:
I'd suggest `Hank Scorpio, hank@globex.com`

Reply:
And `stoepsel@primrose.pasture`

Reply:
Never heard of a guy called Stoepsel...